les/handler.go

```diff
@@ -50,14 +50,22 @@ const (
 
 	ethVersion = 63 // equivalent eth version for the downloader
 
-	MaxHeaderFetch           = 192 // Amount of block headers to be fetched per retrieval request
-	MaxBodyFetch             = 32  // Amount of block bodies to be fetched per retrieval request
-	MaxReceiptFetch          = 128 // Amount of transaction receipts to allow fetching per request
-	MaxCodeFetch             = 64  // Amount of contract codes to allow fetching per request
-	MaxProofsFetch           = 64  // Amount of merkle proofs to be fetched per retrieval request
-	MaxHelperTrieProofsFetch = 64  // Amount of merkle proofs to be fetched per retrieval request
-	MaxTxSend                = 64  // Amount of transactions to be send per request
-	MaxTxStatus              = 256 // Amount of transactions to queried per request
+	// MaxHeaderFetch is the maximum number of block headers to be fetched per retrieval request
+	MaxHeaderFetch = 192
+	// MaxBodyFetch is the maximum number of block bodies to be fetched per retrieval request
+	MaxBodyFetch = 32
+	// MaxReceiptFetch is the maximum number of transaction receipts to allow to be fetched per request
+	MaxReceiptFetch = 128
+	// MaxCodeFetch is the maximum number of contract codes to allow to be fetched per request
+	MaxCodeFetch = 64
+	// MaxProofsFetch is the maximum number of merkle proofs to be fetched per retrieval request
+	MaxProofsFetch = 64
+	// MaxHelperTrieProofsFetch is the maximum number of merkle proofs to be fetched per retrieval request
+	MaxHelperTrieProofsFetch = 64
+	// MaxTxSend is the maximum number of transactions to be sent per request
+	MaxTxSend = 64
+	// MaxTxStatus is the maximum number of transactions to queried per request
+	MaxTxStatus = 256
```

Comment:
I'm not sure the change in this particular block was an improvement... I think it was better before, more clarity

Reply:
This was just the go-lint warnings thing. Without those comments you get lots of warnings in the IDE